demo/app-root.ts

```diff
@@ -1,19 +1,105 @@
-import { html, css, LitElement, customElement } from 'lit-element';
-import '../src/your-webcomponent';
+import { html, css, LitElement, customElement, query } from 'lit-element';
+import '../src/iaux-underlined-tab-bar';
+import type { UnderlinedTabBar } from '../src/iaux-underlined-tab-bar';
 
 @customElement('app-root')
 export class AppRoot extends LitElement {
+  @query('iaux-underlined-tab-bar') tabBar!: UnderlinedTabBar;
+
+  private itemClicked(e: CustomEvent) {
```

Comment:
more naming finesse: this could be named something like `logEventDetails()` to communicate what the method itself does